src/resolvers/addressChange.ts

```diff
@@ -260,3 +260,16 @@ const transformResult = (
     quoteIds,
   }
 }
+
+interface IRequired {
+  (fieldName: string, value?: number): number
+  (fieldName: string, value?: string): string
+  (fieldName: string, value?: boolean): boolean
+}
+
+const requiredField: IRequired = (fieldName: string, value?: any) => {
+  if (value == null) {
+    throw Error(`Field: ${fieldName} is required but was not provided`)
+  }
+  return value
+}
```

Comment:
```suggestion
function requiredField<T>(fieldName: string, value: T | null | undefined) {
  if (!value) {
    throw Error(`Field: ${fieldName} is required but was not provided`)
  }
  return value
}
```

This simpler perhaps? :)

Reply:
Yes! I wanted to do something like this but didn't get it to work 😄

Reply:
Maybe because I was using an arrow function?

Reply:
But I have to use `value == null`, otherwise it fails when `value = false`